packages/terra-clinical-alert/src/Alert.scss

```diff
@@ -0,0 +1,6 @@
+/* stylelint-disable selector-class-pattern  */
+:local {
+  .gap-checking-icon {
+    color: #e50000;
+  }
```

Comment:
There isn't any styling required for outside records besides the styling already provided by the custom alert type.